openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -391,6 +391,57 @@ def _get_all_parents(groups):
     AccountGroup._parent_store_compute()
 
 
+def _create_fixed_vietnam_bank_accounts(env):
+    """Create new accounts in v14"""
+    vn_chart_template = env.ref("l10n_vn.vn_template", raise_if_not_found=False)
+    if vn_chart_template:
+        companies = env["res.company"].search(
+            [("chart_template_id", "=", vn_chart_template.id)]
+        )
+        for company in companies:
+            openupgrade.logged_query(
+                env.cr,
+                """
+                UPDATE account_account
+                SET code = '1125', name = {}
+                WHERE code = '1121';
+                """.format(
+                    _("Bank")
+                ),
+            )
+
+            liquidity_account_type = env.ref(
+                "account.data_account_type_liquidity", raise_if_not_found=False
+            )
+            query = """
+            INSERT INTO account_account
+            ( name, code, user_type_id,
+            company_id, internal_type, internal_group, reconcile)
+            VALUES
+            ({4}, '1121', {0}, {1}, '{2}', '{3}', false)
+            """
+            if not env["account.account"].search(
+                [("code", "=", "1122"), ("company_id", "=", company.id)]
+            ):
+                query += ",({5}, '1122', {0}, {1}, '{2}', '{3}', false)"
+            if not env["account.account"].search(
+                [("code", "=", "1123"), ("company_id", "=", company.id)]
+            ):
+                query += ",({6}, '1123', {0}, {1}, '{2}', '{3}', false)"
+            openupgrade.logged_query(
+                env.cr,
+                query.format(
+                    liquidity_account_type.id,
+                    company.id,
+                    liquidity_account_type.type,
+                    liquidity_account_type.internal_group,
+                    _("Vietnamese Dong"),
+                    _("Foreign currencies"),
+                    _("Monetary Gold"),
```

Comment:
@royleviindoo @tranngocson1996 mình đâu cần hàm `_create_fixed_vietnam_bank_accounts` khi migrate từ 13 lên 14 nhỉ các chú? Hàm này chỉ để tạo sẵn ra các account cho tài khoản ngân hàng, cụ thể: 1121, 1122, 1123. Nói chính xác thì là giữ chỗ cho 3 tài khoản này. Tuy nhiên dữ liệu ở 13 như nào thì lên 14 mình vẫn phải tôn trọng và giữ nguyên lại chứ. Mấy cái tài khoản này là Odoo 14 nó sinh ra khi generate COA mới thôi. Có thể bỏ hẳn cái hàm này đi được không các chú. Hiện tại nó chính là nguyên nhân kế toán trưởng không chấp nhận bị thay đổi `1121 Ngân hàng ACB - 7689888888` thành `1121 Bank`. Mà chắc chắn sau này các kế toán khác họ cũng không chấp nhận bị thay đổi nội dung của tài khoản.

Reply:
@leotranviindoo vaang em co PR để bỏ rồi ạ https://github.com/Viindoo/OpenUpgrade/pull/134
cc: @royleviindoo